### PR TITLE
Zombies break free from handcuffs when transforming

### LINKED ISSUE #10614: [New Feature]: Zombies can break out of cuffs much faster/bite when cuffed
### Describe the feature request

Xenos can instantly break cuffs, and zombies cannot. The new anti-zombie meta is to find anyone with a tumor, cuff them, buckle them, threat completely secured. 

Cryobed was removed because it allowed med to just completely halt a zombie infestation by buckling someone. This is the same but even easier since all you need is 15 cable coil and a chair, and boom, your zombie is unable to infect *anyone*.

Round 47699. Zombie infestation, five people got bitten - Med's reaction ? Cuff them all, buckle them, then remove the festering ooze at a leisurely pace. It's the cheapest possible way of dealing with zombies as it just removes all sense of urgency/danger.

Ideally, zombies should be able to burst out of cuffs/buckles within a second alike xenos, or bite while cuffed like a zombie would - although that'd not fix the issue since they could just be buckled to remove all danger as it'd take 1min for them to be able to move again.

### PR DESCRIPTION
**Pull request by Miliviu**
## About The Pull Request

closes #10614
Now zombies will break free of handcuffs whenever they convert
They also can break off handcuffs fast (like a Hulk human would)

## Why It's Good For The Game

No more metagaming Zombies and then cuffing them!

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://github.com/BeeStation/BeeStation-Hornet/assets/45698448/84ea8624-9a71-4699-88f6-c70a656c51a4

</details>

## Changelog
:cl:
tweak: Zombies now break free from handcuffs when converted
tweak Zombies break faster from handcuffs
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -592,6 +592,9 @@
 		say(pick(";RAAAAAAAARGH!", ";HNNNNNNNNNGGGGGGH!", ";GWAAAAAAAARRRHHH!", "NNNNNNNNGGGGGGGGHH!", ";AAAAAAARRRGH!" ), forced = "hulk")
 		if(..(I, cuff_break = FAST_CUFFBREAK))
 			dropItemToGround(I)
+	else if(dna.species.id == "memezombies")
```
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/human/human.dm]: Make this a species trait instead.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/human.dm (hunk 1 above)
Comment (Miliviu) [reply code/modules/mob/living/carbon/human/human.dm]: Hmkey!

Comment (Miliviu) [reply code/modules/mob/living/carbon/human/human.dm]: https://github.com/BeeStation/BeeStation-Hornet/assets/45698448/e6a6d7f4-3c84-4404-98c9-ea3fe02e51aa



Like this?

**Review by PowerfulBacon — APPROVED** [review]
oh nice, you even managed to de-shitcodify the hulk check too

**Pull request merged**